tools/execution_environments/ee-multicloud-public/requirements.yml

```diff
@@ -50,3 +50,9 @@ collections:
 
 # openstacksdk>=1.0.0
 - name: openstack.cloud
+
+- name: ansible.posix
+  version: 1.3.0
+
+- name: ansible.workshops
+  version: 1.0.18
```

Comment:
same here.

Also please sort alphabetically.

Reply:
@miteshget

Reply:
We won't integrate `ansible.workshops` into the default supported EE.

This is catalog item specific. Plus this collection pulls different version of aws collection:

```diff
    version: 6.4.0
+    version: 3.1.1
```

You will see it in the diff in this PR.

I suggest you start a branch to maintain your own EE for this Catalog Item.

Reply:
Actually, i think you should create a new folder:

*  `tools/execution_environments/ee-aap2`

And update it there.  You'll need a quay repo as well

Reply:
Resolved all above query, crated new directory tools/execution_environments/ee-ansible-workshop with all of the requirements for new ee